app/src/main/res/layout/list_steps.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:layout_height="wrap_content">
+
+    <TextView
+            android:id="@+id/txt_short_description"
+            android:padding="@dimen/card_view_margin"
```

Comment:
@KhinThiriSoe this is not a card view. Why are you using `card_view_margin`?

Reply:
it's also used for card_view

Reply:
do I need to create a new one for this one even same padding?

Reply:
yes because it's a not a card view. Imagine a situation you want to change the margin of all card view, so you change that `card_view_margin`. Then, all the paddings that are using `card_view_margin` will also be changed. The point of declaring dp in dimen resource is lost here

Reply:
fixed it